Tutorials/Cache-Static-Assets/index.md

```diff
@@ -0,0 +1,99 @@
+---
+versionFrom: 9.3.0
+---
+
+# Cache static assets
+
+Example to cache static assets by file extension, but excluding Umbraco BackOffice assets.
+
+```csharp
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Net.Http.Headers;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Umbraco.Cms.Core.Composing;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Core.Hosting;
+using Umbraco.Extensions;
+
+public class StaticFilesComposer : IComposer
+{
+    public void Compose(IUmbracoBuilder builder)
+        => builder.Services.AddTransient<IConfigureOptions<StaticFileOptions>, ConfigureStaticFileOptions>();
+
+    private class ConfigureStaticFileOptions : IConfigureOptions<StaticFileOptions>
+    {
+        private readonly IHostingEnvironment _hostingEnvironment;
+        private readonly GlobalSettings _globalSettings;
+        private static readonly HashSet<string> _fileExtensions = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+        {
+            ".ico",
+            ".css",
+            ".js",
+            ".svg",
+            ".woff2"
+        };
```

Comment:
Consider removing the member variables `_hostingEnvironment` and `_globalSettings` in favor of calculating the back-office path in the constructor.

Also consider renaming the `_fileExtensions` member for clarity and maybe add a comment explaining what it holds.

Lastly it is common practice to have static members at the top of the class.


```suggestion
        // These are the extensions of the file types we want to cache (add and remove as you see fit)
        private static readonly HashSet<string> _cachedFileExtensions = new(StringComparer.OrdinalIgnoreCase)
        {
            ".ico",
            ".css",
            ".js",
            ".svg",
            ".woff2"
        };

        private readonly string _backOfficePath;
```